Formula/n/navidrome.rb

```diff
@@ -0,0 +1,32 @@
+class Navidrome < Formula
+  desc "Modern Music Server and Streamer compatible with Subsonic/Airsonic"
+  homepage "https://www.navidrome.org"
+  url "https://github.com/navidrome/navidrome.git",
+      tag:      "v0.51.1",
+      revision: "6d253225de088e27f9a191d6bcedcb2e68b123fa"
```

Comment:
Do we need a Git clone? Can we use a tarball instead?

Reply:
Fixed